src/engine/server_iv.c

```diff
@@ -1070,7 +1070,7 @@ _iv_op(struct ds_iv_ns *ns, struct ds_iv_key *key, d_sg_list_t *value,
 		 * but in-flight fetch request return IVCB_FORWARD, then queued RPC will
 		 * reply IVCB_FORWARD.
 		 */
-		D_WARN("ns %u retry for class %d opc %d rank %u/%u: " DF_RC "\n", ns->iv_ns_id,
+		D_INFO("ns %u retry for class %d opc %d rank %u/%u: " DF_RC "\n", ns->iv_ns_id,
```

Comment:
yeah, mostly this will help us figure out what happened, since debug is not always enabled when sth happened.

Reply:
i understand that, but this will be in the production server log. so far retry messages that i could are debug, so maybe just the test need to be updated to enable debug logs for this component?
anyway, not requesting a change for now and just asking the reasoning for this, which sounds only for testing purposes?

Reply:
well actually i got confused sorry. this was originally warn, so your change is actually making this better. still think it should be debug, but can be done later.